scripts/bootloader/do_sign.py

```diff
@@ -31,9 +32,9 @@ def parse_args():
     return args
 
 
-if __name__ == '__main__':
+if __name__ == 'main':
     args = parse_args()
-    private_key = SigningKey.from_pem(args.private_key.read())
+        private_key = SigningKey.from_pem(args.private_key.read())
     data = args.infile.read()
     signature = private_key.sign(data, hashfunc=hashlib.sha256)
     args.outfile.write(signature)
```

Comment:
- The conditional check `if __name__ == 'main':` should be corrected to `if __name__ == '__main__':`.
  ```suggestion
  if __name__ == '__main__':
  ```
- The indentation of the line `private_key = SigningKey.from_pem(args.private_key.read())` should be consistent with its surrounding code.
  ```suggestion
    private_key = SigningKey.from_pem(args.private_key.read())
  ```